gateway/enforcer/org.wso2.apk.enforcer/src/main/java/org/wso2/apk/enforcer/util/JWTUtils.java

```diff
@@ -246,6 +246,9 @@ public static SignedJWTInfo getSignedJwt(String accessToken,String organization)
     public static boolean isExpired(String token) {
 
         String[] splitToken = token.split("\\.");
+        if (splitToken.length != 3) {
```

Comment:
how can it reach this line? we have a can authenticate method in apikey and jwt authenticators. there we check whther the token is in right format. i think this check has to be added from that level, invalid tokens shouldn't reach this code

Reply:
this ArrayIndexOutOfBoundsException occurs in the next line intermittently, so the previous validations seems to be unaffected in those intermittent scenarios

Reply:
i think we need to fix from that level too

Reply:
fixed